L1Trigger/L1TGlobal/plugins/L1TGlobalSummary.cc

```diff
@@ -0,0 +1,246 @@
+// L1TGlobalSummary:  Use L1TGlobalUtils to print summary of L1TGlobal output
+//
+// author: Brian Winer Ohio State
+//
+
+#include <fstream>
+#include <iomanip>
+
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+
+#include "DataFormats/L1TGlobal/interface/GlobalAlgBlk.h"
+#include "DataFormats/L1TGlobal/interface/GlobalExtBlk.h"
+
+#include "L1Trigger/L1TGlobal/interface/L1TGlobalUtil.h"
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/MessageLogger/interface/MessageDrop.h"
+
+using namespace edm;
+using namespace std;
+using namespace l1t;
+
+
+// class declaration
+class L1TGlobalSummary : public edm::EDAnalyzer {
```

Comment:
This should become a stream or global analyzer and in case of stream then would need a cache